CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_recharge v1.1.0
+This release aligns with the [Fivetran Recharge Connector August 2025](https://fivetran.com/docs/changelog/2025/august-2025#recharge) updates. [PR #35](https://github.com/fivetran/dbt_recharge/pull/35) includes the following updates:
+
+## Schema/Data Changes
+
+**__ total changes • 0 possible breaking changes**
+| **Data Model/Column** | **Change type** | **Old** | **New** | **Notes** |
+| --------------------- | --------------- | ------- |-------- | --------- |
+| `ORDER` | Source removed | | | Sunset in favor of `ORDERS`, which is now standard across all connections. |
+| [`stg_recharge__discount`](https://fivetran.github.io/dbt_recharge/#!/model/model.zendesk.stg_recharge__discount) | Column removed | `applies_to_id` | | Removed due to inconsistent data types (JSON vs. string) in the source. It can be re-enabled via the `recharge__discount_passthrough_columns` variable. See the [Passing Through Additional Columns](https://github.com/fivetran/dbt_recharge/blob/main/README.md#passing-through-additional-columns) section in the README for usage details. |
```

Comment:
Small adjustment to highlight the variable is new.
```suggestion
| [`stg_recharge__discount`](https://fivetran.github.io/dbt_recharge/#!/model/model.zendesk.stg_recharge__discount) | Column removed | `applies_to_id` | | Removed due to inconsistent data types (JSON vs. string) in the source. It can be re-enabled via the newly added `recharge__discount_passthrough_columns` variable. See the [Passing Through Additional Columns](https://github.com/fivetran/dbt_recharge/blob/main/README.md#passing-through-additional-columns) section in the README for usage details. |
```

Reply:
Added.